components/lcd_touch/esp_lcd_touch_cst816s/include/esp_lcd_touch_cst816s.h

```diff
@@ -41,15 +41,20 @@ esp_err_t esp_lcd_touch_new_i2c_cst816s(const esp_lcd_panel_io_handle_t io, cons
  *
  */
 #define ESP_LCD_TOUCH_IO_I2C_CST816S_CONFIG()             \
-    {                                                    \
+    {                                                     \
         .dev_addr = ESP_LCD_TOUCH_IO_I2C_CST816S_ADDRESS, \
-        .control_phase_bytes = 1,                        \
-        .dc_bit_offset = 0,                              \
-        .lcd_cmd_bits = 8,                              \
-        .flags =                                         \
-        {                                                \
-            .disable_control_phase = 1,                  \
-        }                                                \
+        .on_color_trans_done = 0,                         \
+        .user_ctx = 0,                                    \
+        .control_phase_bytes = 1,                         \
+        .dc_bit_offset = 0,                               \
+        .lcd_cmd_bits = 8,                                \
+        .lcd_param_bits = 0,                              \
+        .flags =                                          \
+        {                                                 \
+            .dc_low_on_data = 0,                          \
+            .disable_control_phase = 1,                   \
+        },                                                \
+        .scl_speed_hz = 0                                 \
```

Comment:
@espzav I don't know how, but we overlooked this  :D :D
https://github.com/espressif/esp-bsp/issues/485

Reply:
oh :-( I am so sorry I overlooked it and we haven't got any HW for runtime testing this touch controller :-( We need it.

Reply:
I will fix it soon